doc/west/repo-tool.rst

```diff
@@ -0,0 +1,270 @@
+.. _west-multi-repo:
+
+Multi-repository management
+###########################
+
+.. note::
+
+   Zephyr is currently not managed as a multi-repo. This page describes an
+   upcoming tool and potential workflow.
+
+West includes a set of commands for working with projects composed of multiple
+Git repositories (a *multi-repo*), similar to Google's `repo
+<https://gerrit.googlesource.com/git-repo/>`_ tool.
+
+The rest of this page introduces multi-repo concepts and gives an overview of
+the multi-repo commands, along with an example workflow.
+
+.. note::
+
+   The multi-repo commands are meant to augment Git in minor ways for
+   multi-repo work, not replace it. For tasks that aren't multi-repo-related,
+   use plain Git commands.
+
+   This page explains what the West multi-repo commands do behind the scenes.
+
+Manifests
+=========
+
+A *manifest* is a `YAML <http://yaml.org/>`_ file that gives the URL and
+revision for each repository in the multi-repo (each *project*), possibly along
+with other information. Manifests are normally stored in a separate Git
+repository.
+
+Running ``west init`` to initialize a West installation will clone the selected
+manifest repository. Running ``west init`` without specifying a manifest
+repository will use `a default manifest repository
+<https://github.com/zephyrproject-rtos/manifest>`_ for Zephyr. A different
+manifest repository can be selected with ``west init --manifest-url URL``.
+
+The format of the West manifest is described by a `pykwalify
+<https://pypi.org/project/pykwalify/>`_ schema, found `here
+<https://github.com/zephyrproject-rtos/west/blob/master/src/west/manifest-schema.yml>`_.
+
+The ``manifest-rev`` branch
+***************************
+
+West creates a branch named ``manifest-rev`` in each project, pointing to the
+commit at the project's revision, as specified in the manifest. The
```

Comment:
`master` is not a commit. Maybe we need to rephrase this to express tracking branches?

Reply:
`manifest-rev` is not a symbolic ref (like `HEAD` -> `refs/heads/master`, when you're not in detached `HEAD` mode), so it does point to a commit (whatever commit the revision resolves to).

I'll see if I can rephrase it a bit.

Reply:
Rephrased it a bit. Is that better?